cogs/strike.py

```diff
@@ -121,7 +121,7 @@ async def no_strike_member_button_callback(self, _: discord.Button, interaction:
         The actual handling of the event is done by the command that sent the view,
         so all that is required is to delete the original message that sent this view.
         """
-        logger.debug("\"No\" button pressed. %s", interaction)
+        logger.debug('"No" button pressed. %s', interaction)
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes